src/receipt/receipt_std.rs

```diff
@@ -169,7 +170,7 @@ impl io::Read for Receipt {
             Self::ScriptResult { result, gas_used } => {
                 let buf = bytes::store_number_unchecked(buf, ReceiptRepr::ScriptResult as Word);
 
-                let result = Word::from(*result);
+                let result: Word = (*result).into();
```

Comment:
Is there any change here?

Reply:
nope, this is some cruft leftover from iterating on how to setup the ScriptExecutionResult.